.github/workflows/centraldb_angular_docker_publish.yaml

```diff
@@ -11,6 +11,7 @@ on:
 env:
   DOCKER_USER: kubeflownotebookswg
   IMG: kubeflownotebookswg/centraldashboard-angular
+  TAG: latest
```

Comment:
let's remove the TAG here and just build/push the image two times like this, in the dedicated step:
```yaml
    - name: Run CentralDashboard-Angular build/push
      run: |
        cd components/centraldashboard-angular
        make docker-build docker-push

        export TAG=latest
        make docker-build docker-push
```

Reply:
Actually, I was thinking the issue with the `latest` tag and I think we should close this here once and for all.

Would something like this work?
```yaml
    - name: Run CentralDashboard-Angular build/push
      run: |
        cd components/centraldashboard-angular
        make docker-build docker-push

    - name: Run CentralDashboard-Angular build/push latest
      if: github.ref == 'refs/heads/master'
      run: |
        export TAG=latest
        make docker-build docker-push
```
https://stackoverflow.com/a/58142412
https://docs.github.com/en/actions/learn-github-actions/contexts#github-context

Reply:
Yes this will work, I've made the appropriate changes